packages/p2p-v2/src/components/AdvertiserName/__tests__/AdvertiserNameBadges.spec.tsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import { APIProvider } from '@deriv/api';
+import { render, screen } from '@testing-library/react';
+import AdvertiserNameBadges from '../AdvertiserNameBadges';
+
+const wrapper = ({ children }: { children: JSX.Element }) => <APIProvider>{children}</APIProvider>;
+let mockUseAdvertiserStats = {
+    data: {
+        isAddressVerified: false,
+        isIdentityVerified: false,
+        totalOrders: 10,
+    },
+    isLoading: true,
+};
+
+jest.mock('../../../hooks', () => ({
+    ...jest.requireActual('../../../hooks'),
+    useAdvertiserStats: jest.fn(() => mockUseAdvertiserStats),
+}));
+
+describe('AdvertiserNameBadges', () => {
+    it('should render loading', () => {
+        mockUseAdvertiserStats = {
+            data: {
+                isAddressVerified: false,
+                isIdentityVerified: false,
+                totalOrders: 20,
+            },
+            isLoading: true,
+        };
```

Comment:
if you want to make it "shorter" with what nada mentioned above, but its up to preference tbh, either way is fine, but since its quite short, i dont think you need to follow with what I suggested, but can be used as a future reference to larger mocked data

```suggestion
            ...mockUseAdvertiserStats,
            data: {
                ...mockUseAdvertiserStats.data,
                totalOrders: 20,
            },
        };
```